### ISSUE TITLE
I am impressed by this

### ISSUE THREAD

**Issue by Metalor696** [Tyzer34/plexMusicPlayer#34]
Not an issue! Just wanted to express myself by saying this is really cool! Nice initiative 😸

**Comment by holmesb** [comment]
Me too.  Plex Music Player is fantastic.  Like having free Amazon Music.

**Comment by Tyzer34** [comment]
Thank you! Although I'm not actively supporting it anymore, it is still nice to hear people finding it useful!

**Comment by pasmer** [comment]
May I ask: Is it still working for you? I’ve tried to deploy it on Alexa dev and I got error

**Comment by holmesb** [comment]
Yes still working.  Alexa, ask plex music to play artist\album\song ...
Loving it.

**Comment by pasmer** [comment]
> Yes still working. Alexa, ask plex music to play artist\album\song ...
> Loving it.

Thanks! Maybe I was wrong in some way. My goal is to translate in Italian.
May I ask you some advice about the deployment?
